test/statement/variable.stmt

```diff
@@ -193,4 +193,14 @@ var variableName = ([
 var variableName = (notDelimited + expression);
 <<<
 var variableName =
-    (notDelimited + expression);
\ No newline at end of file
+    (notDelimited + expression);
+>>> Split all variables if an initializer has a split internally.
+var a = 1, b = [element, element, element, element];
+<<<
+var a = 1,
+    b = [
+      element,
+      element,
+      element,
+      element,
+    ];
```

Comment:
Nice! Sometimes the formatting result is so pleasant to look at compared to the original.

Reply:
The really cool one for me is:

```dart
var longVariableName =
    veryLongFunctionName(
      argument,
      another,
    );
```

The old formatter isn't able to produce that output at all. Instead, it gives you:

```dart
var longVariableName =
    veryLongFunctionName(
  argument,
  another,
);
```

Because it isn't able to change the indentation of subsequent lines based on how it decided to split a previous construct.

Reply:
That's awesome. Love to see it!